spyderlib/plugins/runconfig.py

```diff
@@ -467,8 +474,15 @@ def setup_page(self):
         interpreter_layout.addWidget(self.dedicated_radio)
         interpreter_layout.addWidget(self.systerm_radio)
         
-        wdir_group = QGroupBox(_("Working directory"))
-        wdir_bg = QButtonGroup(wdir_group)
+        general_group = QGroupBox("General settings")
+        command_options_enabled = self.create_checkbox(_("Command line options:"),

```

Comment:
Who are we to judge?  Perhaps they always want to set PYTHONWARNINGS, but only when running in Spyder?